pages/common/zmv.md

```diff
@@ -0,0 +1,28 @@
+# zmv
+
+> Move or rename files matching a specified regex pattern.
```

Comment:
Hmm, it doesn't seem like "regex" is the right terminology. From [this page](https://filipe.kiss.ink/zmv-zsh-rename/):

> First of all, let me get this out of the way: `zmv` does not operate using regular expressions, but it uses [glob](http://zsh.sourceforge.net/Doc/Release/Expansion.html#Glob-Operators) operators to match the files. It's the same pattern you are used to use in the CLI environment but, unfortunately, we can't do things like `[0-9]{4}` to match any sequence of 4 numbers. We actually need to repeat the pattern four times — i.e. `[0-9][0-9][0-9][0-9]` - to get the same result.

And from the [official documentation](http://zsh.sourceforge.net/Doc/Release/User-Contributions.html#Other-Functions), linked below:

> The pattern is always treated as an EXTENDED_GLOB pattern.

So maybe let's change "regex pattern" to "extended glob pattern"?

Reply:
```suggestion
> Move or rename files matching a specified extended glob pattern.
```